.github/mergify.yml

```diff
@@ -0,0 +1,43 @@
+pull_request_rules:
+- actions:
+    merge:
+      strict: smart
+      method: merge
+  name: Automatically merge pull requests
+  conditions:
+  - status-success=bench (8.10.2, ubuntu-latest)
+  - status-success=bench (8.6.5, ubuntu-latest)
+  - status-success=bench (8.8.4, ubuntu-latest)
+  - status-success=nix (default, macOS-latest)
+  - status-success=nix (default, ubuntu-latest)
+  - status-success=test (8.10.3, ubuntu-latest)
+  - status-success=test (8.10.3, windows-latest)
+  - status-success=test (8.10.2, ubuntu-latest)
+  - status-success=test (8.10.1, ubuntu-latest)
+  - status-success=test (8.10.1, windows-latest)
+  - status-success=test (8.8.4, ubuntu-latest)
+  - status-success=test (8.8.3, ubuntu-latest)
+  - status-success=test (8.8.2, ubuntu-latest)
+  - status-success=test (8.6.5, ubuntu-latest)
+  - status-success=test (8.6.5, windows-latest)
+  - status-success=test (8.6.4, ubuntu-latest)
+  - status-success=test (windows-latest, 8.10.2.2)
+
+  - 'status-success=ci/circleci: cabal'
+  - 'status-success=ci/circleci: ghc-8.10.1'
+  - 'status-success=ci/circleci: ghc-8.10.3'
+  - 'status-success=ci/circleci: ghc-8.6.4'
+  - 'status-success=ci/circleci: ghc-8.8.2'
+  - 'status-success=ci/circleci: ghc-8.8.3'
+  - 'status-success=ci/circleci: ghc-8.10.2'
+  - 'status-success=ci/circleci: ghc-8.6.5'
+  - 'status-success=ci/circleci: ghc-8.8.4'
+  - 'status-success=ci/circleci: ghc-default'
```

Comment:
Can we drop this one since it's unclear what `ghc-default` means?

Reply:
`ghc-default` is using the default `stack.yaml`, without version
maybe we should change the name in ci (and here too, but keeping it in any case)

Reply:
I have deleted it in #1152